UEFIExtract/meson.build

```diff
@@ -0,0 +1,18 @@
+executable(
+  'UEFIExtract',
+  sources: [
+    'uefiextract_main.cpp',
+    'ffsdumper.cpp',
+    'uefidump.cpp',
+  ],
+  link_with: [
+    lzma,
+    bstrlib,
+    uefitoolcommon,
+  ],
+  dependencies: [
+    zlib,
+  ],
+  install: true,
+  install_dir: get_option('bindir')
```

Comment:
bindir is already the default for executables. :)

Reply:
TIL, fixing, thanks.